libkbfs/kbfs_ops_test.go

```diff
@@ -357,6 +357,7 @@ func injectNewRMD(t *testing.T, config *ConfigMock) (
 	ops := getOps(config, id)
 	ops.head = makeImmutableRMDForTest(
 		t, config, rmd, fakeMdID(tlf.FakeIDByte(id)))
+	ops.headTrusted = true
```

Comment:
can you try writing a test that makes sure the new code behaves as expected, if an MD server sends down a rekey notification and sends down a fake TLF?

You can probably copy some of the code from `TestKBFSOpsMaliciousMDServerRange`